web/src/components/network/NetworkPage.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) [2023-2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { CardBody, Grid, GridItem } from "@patternfly/react-core";
+import { ButtonLink, CardField, EmptyState, Page } from "~/components/core";
+import ConnectionsTable from "~/components/network/ConnectionsTable";
+import { _ } from "~/i18n";
+import { connectionAddresses } from "~/utils/network";
+import { sprintf } from "sprintf-js";
+import { useNetwork, useNetworkConfigChanges } from "~/queries/network";
+import { PATHS } from "~/routes/network";
+import { partition } from "~/utils";
+
+const WiredConnections = ({ connections, devices }) => {
+  const total = connections.length;
+
+  return (
+    <CardField label={total > 0 && _("Wired")}>
+      <CardBody>
+        {total === 0 && <EmptyState title={_("No wired connections found")} icon="warning" />}
+        {total !== 0 && <ConnectionsTable connections={connections} devices={devices} />}
```

Comment:
when I see this pair of condition, cannot be have simple if here? or ternary as suggested at https://legacy.reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator

Reply:
Sure, kind of left over